examples/07_end_to_end/seismic_waveform.ipynb

```diff
@@ -0,0 +1,1322 @@
+   "cell_type": "markdown",
+   "id": "ccbba367-3252-4c7a-8315-2c5c84993632",
+   "metadata": {},
+   "source": [
+    "Seismic waves are vibrations that travel through the Earth, caused by earthquakes and volcano eruptions, among other causes. Seismic waves are recorded by seismometers located in seismic stations which are operated as a part of larger [networks](https://www.fdsn.org/networks/) of such stations. There are also [data centers](https://www.fdsn.org/datacenters/) that make data available to the public both historically and as a live feed via the [SeedLink](https://docs.fdsn.org/projects/seedlink/en/latest/) protocol. \n",
+    "\n",
+    "The python library [`obspy`](https://docs.obspy.org/index.html) is a convenient tool for processing seismological data, including [waveform data](https://docs.obspy.org/packages/obspy.core.html#waveform-data). It provides clients to help access the data from python as well as signal processing functions that help with filtering and event detection (triggering) based on this time series data. Most routines operate on historical time series data represented as numpy arrays. \n",
+    "\n",
+    "In this notebook, our goal is to show how `csp` can be used to wrap functionality from `obspy` to access both real time and historical waveform data. We also illustrate how the filtering and triggering logic can be re-implemented in a way that applies just as easily to real time streaming data as it does to batch historical data.\n",
+    "\n",
+    "To get started, we load the sample waveform \"Stream\" from obspy (i.e. a group of waveforms), and pull a single \"Trace\" from it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "27252e5f-db99-450b-a8b7-1829021754f5",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "BW.RJOB..EHZ | 2009-08-24T00:20:03.000000Z - 2009-08-24T00:20:32.990000Z | 100.0 Hz, 3000 samples"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "st = read()\n",
+    "tr = st[0]\n",
+    "tr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d52ba99-281d-44a6-967e-2877553f7e9b",
+   "metadata": {},
+   "source": [
+    "The \"id\" of the Trace object is composed of the network id (\"BW\"), the station id (\"RJOB\") and the channel (\"EHZ\"). The channel names are composed of the band code (i.e. frequency, where \"E\" is \"Extremely High Frequency\"), the instrument code (\"H\" is \"High Gain Seismometer\") and the orientation code (i.e. \"Z\" is vertical movement). You can read more about channel naming [here](https://ds.iris.edu/ds/nodes/dmc/data/formats/seed-channel-naming/).\n",
+    "\n",
+    "The trace has two main attributes, \"stats\" which contains metadata such as the start/end time, sampling frequency, etc and \"data\" which is a numpy array of the raw measurements, at equally sampled intervals in the time range."
```

Comment:
What actually are the raw measurements? Positional deviations? Also what units are these measurements in. 
It would also be interesting if you alluded to how these measurements can be used for insights into tremors/earthquakes. We wouldn't do any of that in the example, but I think it would add more interest to the notebook.

Reply:
There is actually a bit of an unanswered units question as the real time and historical units don't seem to align, and I don't know the reason. Maybe @melissawm can reach out to some obspy folks to understand that better. This is good feedback though, and we can put more of that info in the introduction.